<a href="https://colab.research.google.com/github/JitendraAlim/Shrew-Weight-Analysis/blob/master/Shrew_Weight_Analysis.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Shrew Weight Analysis

![Shrew](https://news.wttw.com/sites/default/files/styles/full/public/article/image-non-gallery/Shrew%201%20new_0515.jpg?itok=feSJS7JO "Shrew")

***
# About
Data on greater white-toothed shrews (**Crocidura Russula**) which were caught during two trapping periods in gardens in and around a village in the Swiss Alps. The data were collected by M.Sc. students from the University of Lausanne (Switzerland). The data in the file appear as they were entered in the field. Missing data appear as NA. Pregnant female shrews have been removed from the data set.

**Task**:

1. Test whether there is a difference in weight between male and female greater white toothed
shrews (i.e. do they show sexual dimorphism in body weight).
2. Do juvenile shrews differ from adult shrews in their pattern of sexual dimorphism in terms of body weight?

***
# Importing the Shrew Weight data

In [1]:
# Importing Pandas for Data Management
import pandas as pd

df = pd.read_csv("https://raw.githubusercontent.com/JitendraAlim/Shrew-Weight-Analysis/master/Shrew%20Weights.csv")
df.head()

Session   Sex  Class  Weight
0        2  male  adult    13.0
1        1     M   ad/j     8.5
2        1     F  adult    11.5
3        1     M  adult    10.3
4        2     M     ad    13.5

***
# Exploring the data

In [2]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 811 entries, 0 to 810
Data columns (total 4 columns):
 #   Column   Non-Null Count  Dtype  
---  ------   --------------  -----  
 0   Session  811 non-null    int64  
 1   Sex      811 non-null    object 
 2   Class    809 non-null    object 
 3   Weight   797 non-null    float64
dtypes: float64(1), int64(1), object(2)
memory usage: 25.5+ KB


In [3]:
df.shape

(811, 4)

There are 811 observations on shrews based on 4 variables